iphone/TitaniumKit/TitaniumKit.xcodeproj/project.pbxproj

```diff
@@ -1010,7 +1010,7 @@
 					"$(PROJECT_DIR)/TitaniumKit/Libraries/APSAnalytics",
 					"$(PROJECT_DIR)/TitaniumKit/Libraries/APSHTTPClient",
 				);
-				ONLY_ACTIVE_ARCH = NO;
+				ONLY_ACTIVE_ARCH = YES;
```

Comment:
Is this change intended?

Reply:
Yes. This is for 'Debug' mode. A minor change also required in scheme of Distribution target. Updated that as well.